subprojects/platform-play/platform-play.gradle

```diff
@@ -28,3 +41,15 @@ useTestFixtures(project: ':platformBase')
 
 useClassycle()
 strictCompile()
+
+task integTestPrepare(type: org.gradle.testing.IntegrationTest) {
```

Comment:
I went ahead and merged everything. We can reconsider if it effectively becomes unmanageable.